src/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -260,40 +260,4 @@ private void writeDelegates(final GameData data, final ObjectOutputStream out) t
     out.writeObject(DELEGATE_LIST_END);
   }
 
-  /**
```

Comment:
Could this one be moved to a unit test? Essentially just test to make sure a save game loads properly?